app/build.gradle

```diff
@@ -33,4 +34,7 @@ dependencies {
     testCompile 'junit:junit:4.12'
 
     compile 'com.android.support:appcompat-v7:23.4.0'
+    compile 'com.github.markrjr:Salut:v0.5.0'
```

Comment:
Salut is licensed under MIT and LoganSquare under Apache 2.0

I need to include LoganSquare because Salut uses it. I would rather use Gson, as I need to do a workaround at the moment (you can find it where I parse the data: LoganSquare.parse(.....))